tools/labs/Makefile

```diff
@@ -31,6 +33,24 @@ copy: $(YOCTO_IMAGE)
 	sudo umount $(TEMPDIR)
 	rmdir $(TEMPDIR)
 
+scp: ip
+	scp -r skels root@$(shell cat .ip):
+
+ssh: ip
+	ssh root@$(shell cat .ip)
+
+ip:
+	if [ ! -e qemu.mon ]; then exit 1; fi
+	GUEST_IP=$(GUEST_IP_CMD); \
+	[ "$$GUEST_IP" = "arp:" ] && echo -n "Waiting for machine"; \
+	while [ "$$GUEST_IP" = "arp:" ]; do \
+		echo -n "."; \
+		sleep 1; \
+		GUEST_IP=`arp -e -n -i tap0 | tail -1 | cut -d ' ' -f 1`; \
+	done; \
+	echo ""; \
+	echo -n $$GUEST_IP > .ip
```

Comment:
Why make this ip file hidden? Why not just output the IP value?

Reply:
I extracted the ip target so I don't have duplicate code for finding the ip in the scp and ssh targets. Saving the ip in a hidden file was the only way I could think of making it accesible to the other targets. Do you have any suggestions about this?

Reply:
You can do a `make -s ip` to get the output.

Reply:
@VladNastase You could add a script in tools/labs/qemu to get the ip address and call that for the scp and ssh targets, e.g.

```
ssh:
    IP=$$(qemu/get_vm_ip.sh); if [ -n "$IP" ]; then ssh ...
```